core-web/libs/sdk/client/README.md

```diff
@@ -169,237 +233,280 @@ const { pageAsset, content } = await client.page.get('/about-us', {
+                  title
+                  target
+                  type
+                  children {
+                      hash
+                      href
+                      title
+                      target
+                      type
+                  }
+              }
             `
-        ]
+        }
     }
 });
+```
 
-// Access page data
-console.log(pageAsset.containers);
+#### Method Signature:
 
-// Access content data
-console.log(content.blogPosts);
+```typescript
+get<T extends DotCMSExtendedPageResponse = DotCMSPageResponse>(
+    url: string,
+    options?: DotCMSPageRequestParams
+): Promise<DotCMSComposedPageResponse<T>>;
 ```
 
-### Fetching Navigation
+### Content API
 
-The new API allows you to fetch navigation data using GraphQL through the page.get method:
+**Overview:**
+The Content API allows you to fetch content collections from dotCMS. It provides a builder pattern for constructing complex queries and filtering content based on various criteria.
 
-```javascript
-// Fetch navigation using the page.get method with GraphQL
-const { content } = await client.page.get('/', {
-    languageId: '1',
-    graphql: {
-        content: {
-            nav: `
-                query {
-                    nav {
-                        identifier
-                        path
-                        label
-                        children {
-                            identifier
-                            path
-                            label
-                        }
-                    }
-                }
-            `
-        }
-    }
-});
+The `getCollection` method uses immutable patterns, so each method call returns a new instance. This means you can reuse partial queries by storing them before adding additional filters.
 
-// Access navigation data
-console.log(content.nav);
-```
+**Builder Methods:**
 
-> **Note:** The legacy `client.nav.get()` method is still available but deprecated.
+| Method       | Arguments                | Description                        |
+| ------------ | ------------------------ | ---------------------------------- |
+| `query()`    | `string` \| `BuildQuery` | Filter content using query builder |
+| `limit()`    | `number`                 | Set page size                      |
+| `page()`     | `number`                 | Set page number                    |
+| `sortBy()`   | `SortBy[]`               | Order results                      |
+| `language()` | `number \| string`       | Set content language               |
 
-#### Legacy Navigation Example
+**Usage:**
 
-```javascript
-// Legacy approach - still works but is deprecated
-const navData = await client.nav.get({
-    path: '/',
-    depth: 2,
-    languageId: 1
-});
+```typescript
+const blogs = await client.content
+    .getCollection('Blog')
+    .query((qb) =>
+        qb.field('title').contains('dotCMS').and().field('publishDate').greaterThan('2023-01-01')
+    )
+    .limit(10)
+    .page(1)
+    .fetch();
```

Comment:
Check the [tests](https://github.com/dotCMS/core/blob/316d084c0c214c259a87d0048893a5f592abd2be/core-web/libs/sdk/client/src/lib/client/content/builders/collection/collection.spec.ts) this code is not right